crates/edr_napi/Cargo.toml

```diff
@@ -18,8 +18,8 @@ k256 = { version = "0.13.1", default-features = false, features = ["arithmetic",
 log = { version = "0.4.20", default-features = false }
 # when napi is pinned, be sure to pin napi-derive to the same version
```

Comment:
These don't seem to be released in tandem (anymore?). Should we remove the comment or reword it to something like "If you bump the napi version, please also make sure that we use a compatible napi-derive"?.

This should ideally be handled by the upstream package but sometimes mistakes happen and I assume one of these were the reason for this warning, here.

Reply:
Feel free to. I also removed the comment in `feat/multichain`